dist/index.html

```diff
@@ -0,0 +1,306 @@
+          <li class="nav__item">
+            <a href="#Laptops&#38;Computers" data-qa="nav-hover" class="nav__link">
+            Laptops &#38; Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Gadgets" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Tablets" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Photo" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="page__main">
+      <section class="page__card">
+        <article data-qa="card" class="card">
+          <img src="/imac.7817668a.jpeg" alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)" class="card__logo">
+
+          <h2 class="card__description">
+             APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="card__stars stars">
+            <div class="stars__container stars__container--4">
+              <div class="stars__container-star"></div>
+              <div class="stars__container-star"></div>
+              <div class="stars__container-star"></div>
+              <div class="stars__container-star"></div>
+              <div class="stars__container-star"></div>
+            </div>
+
+            <p class="stars__reviews-text"> Reviews: 5</p>
+          </div>
+
+          <div class="card__prices">
+            <p class="card__text">Price:</p>
+            <p class="card__amount">$2,199</p>
+          </div>
+
+          <a data-qa="card-hover" class="card__link-buy" href="/">
+            BUY
+          </a>
+        </article>
+        <article class="card">
+          <img src="/imac.7817668a.jpeg" alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)" class="card__logo">
+
+          <h2 class="card__description">
+             APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="card__stars stars">
+            <div class="stars__container stars__container--4">
+              <div class="stars__container-star"></div>
+              <div class="stars__container-star"></div>
+              <div class="stars__container-star"></div>
+              <div class="stars__container-star"></div>
+              <div class="stars__container-star"></div>
+            </div>
+
+            <p class="stars__reviews-text"> Reviews: 5</p>
+          </div>
+
+          <div class="card__prices">
+            <p class="card__text">Price:</p>
+            <p class="card__amount">$2,199</p>
+          </div>
+
+          <a class="card__link-buy" href="/">
+            BUY
+          </a>
+        </article>
+        <article class="card">
```

Comment:
add empty lines between siblings
```suggestion
        </article>
        
        <article class="card">
```

Reply:
@Oleksandr Polotniuk
 answer me

Hello, it's not critical, everything looks ok, it's just that maybe the pixel got lost here, or a bug with tests started.